src/resources/marketplace.js

```diff
@@ -217,6 +217,7 @@ class Marketplace {
       ipfsBytes,
       {
         price,
+        commission: offerData.commission,
```

Comment:
Everything passed in offerData gets stored in IPFS. so if we pass the commission this way, let's use a Money object with {currency, amount). and then call the contract with
`commission: offerData.commission.amount`

An alternative would be that the  DApp does not pass the commission. It is up to origin-js to set it before calling the contract. This method would load the listing ipfs data and get commission from there. When we implement multi-unit and fractional usage, I think we might have to do that anyway to calculate the commission (for example to split it in the case of multi-units so that it gets spread over N units ?).

Super nit: move commission one line down to keep price and currency together since they are related.